docs/create_and_deploy_jh_image.md

```diff
@@ -0,0 +1,160 @@
+# How to create a Jupyterhub image and deploy it on ODH
+
+This howto is about getting your repository built into an image and having that image made available in (your) Jupyterhub instance.
+
+We are going to deal with 2 issues:
+
+ 1. Have an image built and added to a public image repository.
+ 1. Add your image to an ODH instance (that has a Jupyterhub in it).
```

Comment:
I missed that it's sr no. 1. here. Can you change that to 2 please?

Reply:
writing it as "1." every time triggers the automatic numbering feature of markdown

so this in intended and it correctly renders as "1." and "2." (see the link in description)

Reply:
Got it! @martinpovolny 🙂